services/web/server/tests/unit/with_dbs/conftest.py

```diff
@@ -132,6 +135,17 @@ def postgres_db(app_cfg, postgres_service):
 @pytest.fixture
 def web_server(loop, aiohttp_server, app_cfg, monkeypatch, postgres_db):
     app = create_application(app_cfg)
+
+    async def return_session(request: web.Request):
+        session = await get_aiohttp_session(request)
+        return web.json_response(dict(session))
+    app.router.add_get("/session", return_session)
+    async def delete_user_email(request: web.Request):
+        session = await get_aiohttp_session(request)
+        del session["user_email"]
+        return web.HTTPOk()
+    app.router.add_get("/delete_user_email", delete_user_email)
+
```

Comment:
Create a specific fixture in your module. This fixture is used in MOST OF THE tests within this folder!